java/src/jmri/implementation/AbstractIdTag.java

```diff
@@ -29,6 +31,8 @@
 
     protected Date whenLastSeen = null;
 
+    protected ConcurrentHashMap<String,Object> properties = null;
```

Comment:
Isn't this a reimplementation of something that's already in AbstractNamed Bean?